### PR TITLE
User/athanaseus/katcp read the docs

### PR DESCRIPTION
**Pull request by Athanaseus**
http://katcp-python.readthedocs.io/en/latest/

### CODE REVIEW

**Comment by tockards** [comment]
:+1:

File: doc/releasenotes.rst [hunk 1]
```diff
@@ -286,4 +286,23 @@ your git origin.
...
+
+  1. Create an account on Read the Docs. You will get an email verifying your
+     email address which you should accept within 7 days.
+  2. Log in and click on "Import".
+  3. Give your project a name, add the HTTPS link for your GitHub project, and
+     select Git as your repository type.
+  4. Fill in the rest of the form as needed and click "Create".
+  5. The click "Build"
+
+* Automatically updated when you push to GitHub
```
Comment (fjoubert) [inline doc/releasenotes.rst]: This is great, no more manually uploading docs like cavemen.

File: doc/releasenotes.rst [hunk 2]
```diff
@@ -286,4 +286,23 @@ your git origin.
...
+
+  6. On GitHub, navigate to your repository and click on "Settings".
+  7. In the sidebar, click on "Web Hooks & Services", then find and click on
```
Comment (fjoubert) [inline doc/releasenotes.rst]: This is actually called `Integrations & Services`

File: README.rst [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
 Python client and server library for the Karoo Array Telescope Control Protocol
 (KATCP).  For more info, see the doc directory in this repository, or for online
 documentation of the latest release, `read the katcp-python documentation
 <http://katcp-python.readthedocs.io/en/latest/>`_.
+
+.. |Doc Status| image:: https://readthedocs.org/projects/katcp-python/badge/?version=latest
+                :target: http://katcp-python.readthedocs.io/en/latest
+                :alt:
+
+.. |Pypi Version| image:: https://img.shields.io/pypi/v/katcp.svg
```
Comment (bmerry) [inline README.rst]: Pypi -> PyPI ?

File: doc/releasenotes.rst [hunk 4]
```diff
@@ -286,4 +286,23 @@ your git origin.
...
 
+* Host package docs on `http://readthedocs.org`
+
+  1. Create an account on Read the Docs. You will get an email verifying your
+     email address which you should accept within 7 days.
+  2. Log in and click on "Import".
+  3. Give your project a name, add the HTTPS link for your GitHub project, and
+     select Git as your repository type.
+  4. Fill in the rest of the form as needed and click "Create".
+  5. The click "Build"
```
Comment (bmerry) [inline doc/releasenotes.rst]: Typo "The click"

**Pull request merged**